lib/dalli/protocol/value_marshaller.rb

```diff
@@ -19,7 +19,7 @@ class ValueMarshaller
 
       OPTIONS = DEFAULTS.keys.freeze
 
-      def_delegators :@value_serializer, :serializer
+      def_delegators :@value_serializer, :serializer, :raw_by_default?
```

Comment:
You don't need this method here, delegators are used when this class (the value marshaller) wants to use `@value_serializer.raw_by_default?` but you don't want to always type out `@value_serializer`